src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -59,10 +58,8 @@ file that was distributed with this source code.
     <script>
         {% autoescape 'js' %}
         jQuery(function ($) {
-            // Select2 v3 does not used same input as v4.
-            // NEXT_MAJOR: Remove this BC layer while upgrading to v4.
```

Comment:
I think these majors in JS refer to Select v4, not the branch. I'll try to review completely on the weekend.

Reply:
I will revert those changes, you are right, we are using select v3 in our assets